engine/src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -51,15 +51,30 @@ trait DataAccess extends AutoCloseable {
   // TODO needed to support compatibility with current code, this seems like an inefficient way of getting
   // TODO workflow outputs.
   /** Returns all outputs for this workflowId */
-  def getWorkflowOutputs(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
+  def getAllWorkflowOutputs(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
 
-  def getAllOutputs(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
+  def getWorkflowOutputs(workflowId: WorkflowId)
+                        (implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]] = {
+    // TODO: Filtering should be done in the database
+    getAllWorkflowOutputs(workflowId) map { _ filterNot ExecutionDatabaseKey.isCallLog }
```

Comment:
At this point I feel like we might as well wait the couple of weeks for the underscore class prior to doing too deep a dive into slick stuff